pkg/components/metallb/manifests.go

```diff
@@ -816,3 +816,42 @@ data:
       "version": 1
     }
 `
+
+const metallbPrometheusRule = `
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: alertmanager-rules
+  namespace: metallb-system
+  labels:
+    release: prometheus-operator
+    app: prometheus-operator
+spec:
+  groups:
+  - name: metallb-rules
+    rules:
+    - alert: MetalLBNoBGPSession
+      expr: metallb_bgp_session_up != 1
+      for: 2m
+      annotations:
+        description: '{{ $labels.instance }}: MetalLB has not established a BGP session for more than 2 minutes.'
+        summary: '{{ $labels.instance }}: MetalLB hasnt established BGP session.'
```

Comment:
```suggestion
        summary: '{{ $labels.instance }}: MetalLB hasn no established BGP session.'
```

Reply:
Can I do suggestion for your suggestion? :smile: There is a typo :see_no_evil:

Reply:
I don't think that works, I'll edit :joy: